scrapers/Traxxx.py

```diff
@@ -0,0 +1,83 @@
+import re, sys, copy, json
+import requests
+
+import py_common.log as log
+from traxxx_interface import TraxxxInterface
+
+def main():
+  global traxxx
+
+  mode = sys.argv[1]
+  traxxx = TraxxxInterface()
+  fragment = json.loads(sys.stdin.read())
+
+  data = None
+
+  log.info(mode)
+
+  if mode == 'scene_name':
+    data = scene_name(fragment)
+  if mode == 'scene_query_fragment':
+    data = scene_query_fragment(fragment)
+  if mode == 'scene_fragment':
+    data = scene_fragment(fragment)
+
+
+  if mode == 'performer_lookup':
+    data = performer_lookup(fragment)
+  if mode == 'performer_fragment':
+    data = performer_fragment(fragment)
+  if mode == 'performer_url':
+    data = performer_url(fragment)
+
+
+  if data:
+    # log.info(json.dumps(data))
+    print(json.dumps(data))
+  else:
+    print("")
+
+# Return a list of scenes from a search
+def scene_name(fragment):
+  title = None
+  if not title:
+    title = fragment.get("title")
```

Comment:
isnt the above the same as 
```py
def scene_name(fragment):
  title = fragment.get("title")
```
?

Reply:
yeah think I just wanted to be able to re organize the blocks in future if needed without changing any logic, I updated it as it is most likely unnecessary